src/rfac/crfrmin.c

```
@@ -214,31 +216,39 @@ FILE *out_stream;
-         strcpy(r_name,"Rb");
+         snprintf(r_name, sizeof(r_name), "%s", "Rb");
        }
 
 /* Calculate sum of theor. intensities for normalization */
        norm = 0.;
        for(i_leng = 0; i_leng < n_leng; i_leng++)
          norm += t_int[i_leng];
 
-       if(i_list < 9) sprintf(linebuffer,"%s.0%dt",args->iv_file, i_list+1);
-       else           sprintf(linebuffer,"%s.%dt", args->iv_file, i_list+1);
+       if (i_list < 9) {
```

Clang (14.0.6)

[warning] if with identical then and else branches

(bugprone-branch-clone)